mne/viz/topomap.py

```diff
@@ -221,48 +230,57 @@ def plot_projs_topomap(projs, layout=None, cmap=None, sensors=True,
     if len(axes) != len(projs):
         raise RuntimeError('There must be an axes for each picked projector.')
     for proj_idx, proj in enumerate(projs):
-        axes[proj_idx].set_title(proj['desc'][:10] + '...')
+        title = proj['desc']
+        title = '\n'.join(title[ii:ii + 22] for ii in range(0, len(title), 22))
+        axes[proj_idx].set_title(title, fontsize=10)
         ch_names = _clean_names(proj['data']['col_names'],
                                 remove_whitespace=True)
         data = proj['data']['data'].ravel()
 
-        idx = []
-        for l in layout:
-            is_vv = l.kind.startswith('Vectorview')
-            if is_vv:
-                from ..channels.layout import _pair_grad_sensors_from_ch_names
-                grad_pairs = _pair_grad_sensors_from_ch_names(ch_names)
-                if grad_pairs:
-                    ch_names = [ch_names[i] for i in grad_pairs]
-
-            l_names = _clean_names(l.names, remove_whitespace=True)
-            idx = [l_names.index(c) for c in ch_names if c in l_names]
+        if isinstance(layout, Info):
```

Comment:
I don't like that. Please add an explicit info parameter and require that layout=None if info is not None